src/frontend/src/scheduler/plan_fragmenter.rs

```diff
@@ -52,7 +54,7 @@ pub const ROOT_TASK_OUTPUT_ID: u32 = 0;
 pub type TaskId = u32;
 
 /// Generated by [`BatchPlanFragmenter`] and used in query execution graph.
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Serialize)]
 pub struct ExecutionPlanNode {
```

Comment:
I think according to our discussion, it's better to use customized serialization for `ExecutionPlanNode`.